src/objects/options_obj.f90

```diff
@@ -1613,41 +1616,50 @@ end subroutine cu_parameters_namelist
     !!
     !! -------------------------------
-    subroutine set_default_LU_categories(urban_category, ice_category, water_category, LU_Categories)
+    subroutine set_default_LU_categories(options, urban_category, ice_category, water_category, LU_Categories, lake_category)
         ! if various LU categories were not defined in the namelist (i.e. they == -1) then attempt
         ! to define default values for them based on the LU_Categories variable supplied.
         implicit none
-        integer, intent(inout) :: urban_category, ice_category, water_category
+        type(options_t),    intent(inout)::options
+        integer, intent(inout) :: urban_category, ice_category, water_category, lake_category
         character(len=MAXVARLENGTH), intent(in) :: LU_Categories
 
         if (trim(LU_Categories)=="MODIFIED_IGBP_MODIS_NOAH") then
             if (urban_category==-1) urban_category = 13
             if (ice_category==-1)   ice_category = 15
             if (water_category==-1) water_category = 17
+            if (lake_category==-1) lake_category = 21
 
         elseif (trim(LU_Categories)=="USGS") then
             if (urban_category==-1) urban_category = 1
             if (ice_category==-1)   ice_category = -1
             if (water_category==-1) water_category = 16
+            ! if (lake_category==-1) lake_category = 16  ! No separate lake category!
+            if((options%physics%watersurface==3) .AND. (this_image()==1)) then
+                write(*,*) "WARNING: Lake model selected (water=3), but USGS LU-categories has no lake category"
+            endif
 
         elseif (trim(LU_Categories)=="USGS-RUC") then
             if (urban_category==-1) urban_category = 1
             if (ice_category==-1)   ice_category = 24
             if (water_category==-1) water_category = 16
+            if (lake_category==-1) lake_category = 28
             ! also note, lakes_category = 28
-            write(*,*) "WARNING: not handling lake category (28)"
+            ! write(*,*) "WARNING: not handling lake category (28)"
 
         elseif (trim(LU_Categories)=="MODI-RUC") then
             if (urban_category==-1) urban_category = 13
             if (ice_category==-1)   ice_category = 15
             if (water_category==-1) water_category = 17
+            if (lake_category==-1) lake_category = 21
             ! also note, lakes_category = 21
-            write(*,*) "WARNING: not handling lake category (21)"
+            ! write(*,*) "WARNING: not handling lake category (21)"
 
         elseif (trim(LU_Categories)=="NLCD40") then
             if (urban_category==-1) urban_category = 13
             if (ice_category==-1)   ice_category = 15 ! and 22?
-            if (water_category==-1) water_category = 17 ! and 21
+            ! if (water_category==-1) water_category = 17 ! and 21 'Open Water'
+            if(options%physics%watersurface==3) write(*,*) "WARNING: Lake model selected (water=3), but NLCD40 LU-categories has no lake category"
```

Comment:
Why not make this `== kWATER_LAKE` ?

Reply:
kWATER_LAKE=3, ISLAKE =options%lsm_options%lake_category, and is based on the above.

Reply:
Yes, I just mean, don't hard code "3" here, use kWATER_LAKE so that if that value ever changes, this won't break.

Reply:
sorry, yes, it didnt read the last line of the code b/c of the scroll bar and was confused with the water_category... time to call it a day ;)